src/chainparams.cpp

```diff
@@ -67,7 +67,7 @@ class CMainParams : public CChainParams {
         consensus.nSubsidyHalvingInterval = 301107;
         consensus.BIP16Height = 0; 
         consensus.BIP34Height = 0;
-        consensus.BIP34Hash = uint256();
+        consensus.BIP34Hash = uint256S("0x46ca17415c18e43f5292034ebf9bbd10de80a61fc6dc17180e6609f33d3b48f3");
```

Comment:
I believe this is the hash of the block that corresponds to when BIP34 is activated, or block 4264.

Reply:
> I believe this is the hash of the block that corresponds to when BIP34 is activated, or block 4264.

Yes correct, it is the block hash of height 0 (the genesis) when BIP34 has been activated.